docs/faq.md

```diff
@@ -137,3 +137,7 @@ the [Architecture documentation](./architecture.md#docker-networking)
 The user interface is based in the open-sourced [Apache Superset](http://superset.apache.org),
 so you can also refer to [Superset tutorials](http://superset.apache.org/tutorial.html)
 for advanced usage of the web interface.
+
+## I Get IOError: [Errno 13] Permission denied: u'./.env'
```

Comment:
This link may not work both in git and GotBook. Could you try in GitBook?
What if using something like the following instead?
```suggestion
## I Get IOError Errno 13 Permission denied
```